lnwallet/channel.go

```diff
@@ -2127,6 +2155,38 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 				return nil, err
 			}
 		}
+
+		// At this point, we'll check to see if we need any extra
+		// resolution data for this output.
+		resolveReq := ResolutionReq{
```

Comment:
Not sure I understand what you mean? The `if` block is done at this point, so this would only not be executed if an error occurred (or `ourAmt < int64(chanState.RemoteChanCfg.DustLimit)` from the outer condition).

Reply:
Yeah this is the case where we've already confirmed on chain, and want to create the resolution blob needed to sweep the output into our wallet.